webpack.config.example.js

```diff
@@ -32,7 +32,8 @@ const path = require('path');
  * Mapbox GL JS uses the webworkify module by default, and that module
  * does things very specific to browserify's module loading system. In this
  * configuration, we alias webworkify to webworkify-webpack to add webpack
- * support.
+ * support. NOTE: there is a known caveat when using 'eval'-related devtool
+ * settings, make sure to double-check your configuration.
```

Comment:
Apologies for the confusion, I'm afraid English is not my first language. I was referring to this part of `webworkerify-webpack`'s README:

> While developing make sure not to set webpack's devtool option to be with some eval configuration because from 1.1.0 version it won't work